src/theory/arith/nl/nonlinear_extension.h

```diff
@@ -224,6 +225,12 @@ class NonlinearExtension
    */
   void sendLemmas(const std::vector<NlLemma>& out);
 
+  /** Run the strategy for the given effort. */
```

Comment:
Carry over the related documentation from checkLastCall and update it as needed.